app/src/main/res/layout/fragment_poi_info.xml

```diff
@@ -7,14 +7,41 @@
     tools:context=".poi.PoiInfoFragment">
 
```

Comment:
You have hardcoded strings and missing content descriptions in the resource files